repos/system_upgrade/el8toel9/models/firewalldusedobjectnames.py

```diff
@@ -0,0 +1,14 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemInfoTopic
+
+
+class FirewalldUsedObjectNames(Model):
+    '''
+    This model contains lists of firewalld object (e.g. zones, services) names
+    in use by the permanent firewalld configuration.
+    '''
```

Comment:
@erig0 use please double-quotes instead of single-quotes for docstrings (everywhere). e.g.
```suggestion
    """
    This model contains lists of firewalld object (e.g. zones, services) names
    in use by the permanent firewalld configuration.
    """
```

Reply:
Done